opmd_viewer/openpmd_timeseries/interactive.py

```diff
@@ -39,21 +39,20 @@ def slider(self, figsize=(10, 10), **kw):
         # Define useful functions
         # -----------------------
 
-        def refresh_field(sender_change=None, force=False):
+        def refresh_field(change, force=False):
```

Comment:
let us keep `change=None` so we can still call it without the object that we do not use anyway